### PR TITLE
Decaying Range Sensor Layer Cost

### PR DESCRIPTION
**Pull request by abolinsky**
Plus fixing some ugly formatting

### CODE REVIEW

**Comment by hendrik-skubch** [comment]
Your formatting is against our style, where does that come from?

**Comment by hendrik-skubch** [comment]
Your formatting is against our style, where does that come from?

File: range_sensor_layer/src/range_sensor_layer.cpp [hunk 1]
```diff
@@ -382,51 +391,58 @@ void RangeSensorLayer::updateBounds(double robot_x, double robot_y, double robot
...
+
+      if (prob == costmap_2d::NO_INFORMATION) {
         it++;
         continue;
-      }
-      else if(prob>mark)
+      } else if (prob > mark) {
         current = costmap_2d::LETHAL_OBSTACLE;
-      else if(prob<clear)
+        costmap_[it]--;
```
Comment (hendrik-skubch) [inline range_sensor_layer/src/range_sensor_layer.cpp]: use prefix operators unless you need the postfix semantics.

Comment (abolinsky) [reply range_sensor_layer/src/range_sensor_layer.cpp]: done

**Comment by abolinsky** [comment]
> Your formatting is against our style, where does that come from?

Formatted using io-projects' clang format

**Review by hendrik-skubch — APPROVED** [review]

**Pull request merged**